src/blacklist.js

```diff
@@ -0,0 +1,12 @@
+const Url = require("url");
```

Comment:
Curious that Prettier didn't catch this. It should be single quotes. 
Perhaps the config for Prettier is wrong.